docs/documentation/nodes.md

```diff
@@ -1,42 +1,45 @@
 # Observables & Interaction
 
 Interaction and animations in Makie are handled using [`Observables.jl`](https://juliagizmos.github.io/Observables.jl/stable/).
-`Observable`s are called `Node`s in Makie for historical reasons and the two terms are used interchangeably.
 An `Observable` is a container object whose stored value you can update interactively.
 You can create functions that are executed whenever an observable changes.
 You can also create observables whose values are updated whenever other observables change.
 This way you can easily build dynamic and interactive visualizations.
 
-On this page you will learn how the `Node`s pipeline and the event-based interaction system work.
+On this page you will learn how the `Observable`s pipeline and the event-based interaction system work. Besides this, there is also a video tutorial on how to make interactive visualizations (or animations) with Makie.jl and the `Observable` system:
 
-## The `Node` structure
+```@raw html
```

Comment:
@jkrumbiegel do you know how to properly inline this with Franklin?

Reply:
I have seen you guys have already put the tutorial in the home page under tutorials, so maybe we can just crosslink there? (I don't know how)

Reply:
Cross-linking would also work, although there isn't the perfect thing to link to I guess